kernel/src/actions/mod.rs

```diff
@@ -128,6 +131,11 @@ impl Protocol {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Schema)]
+pub struct CommitInfo {
+    pub kernel_version: Option<String>,
```

Comment:
do we want to include the field that spark usually writes? maybe not until we have a use case.

Reply:
hm. I think there are _many_ fields spark usually writes and I didn't want to open that can of worms until we have a reason. thoughts?

Reply:
Yep, that's fine, we can start small :)